crates/superposition/Superposition.cac.toml

```diff
@@ -1,9 +1,28 @@
 [default-config]
 mandatory_dimensions = { "value" = [
 ], "schema" = { "type" = "array", "items" = { "type" = "number" } } }
+experiments_webhook_config = { "value" = { "enabled" = false }, "schema" = { "type" = "object", "properties" = { "enabled" = { "type" = "boolean" }, "configuration" = { "type" = "object", "properties" = { "url" = { "type" = "string" }, "method" = { "enum" = [
+    "Post",
+    "Get",
+], "type" = "string" }, "headers" = { "type" = "array", "items" = { "type" = "string", "enum" = [
+    "ConfigVersion",
+] } }, "authorization" = { "type" = "object", "properties" = { "key" = { "type" = "string" }, "value" = { "type" = "string" } }, "required" = [
+    "key",
+    "value",
+] }, "required" = [
+    "url",
+    "method",
+    "headers",
+    "authorization",
+], "additionalProperties" = false } } }, "required" = [
+    "enabled",
+] } }
 
 [dimensions]
 tenant = { schema = { "type" = "string", "enum" = ["test", "dev"] } }
 
 [context."$tenant == 'dev'"]
 mandatory_dimensions = []
+# experiments_webhook_config = { "enabled" = true, "configuration" = { "url" = "http://localhost:8080/config/test", "method" = "Get", "headers" = [
```

Comment:
lets have it added as with "enabled" = false and the config added, will ensure things are getting validated in local